### DISCUSSION TITLE
per AS top IP addresses

### DISCUSSION THREAD

**Discussion by Rico29** [akvorado/akvorado#188] [Q&A]
Hello
Is there a way to visualize, for a given ASN, the top talkers IP addresses ?
if not, would be very useful.
Regards

**Comment by vincentbernat** [comment]
Filter on `SrcAS = xxxx`, Dimension `SrcAddr`.

**Comment by Rico29** [comment]
It would be a great thing to be able to click on items in "sankey" graph to see details of top flows concering the clicked item.
